types.d.ts

```diff
@@ -76,6 +76,8 @@ export interface BlogSettings {
   port?: number;
   /** The hostname to serve the blog on */
   hostname?: string;
+  /** When true, readtime will not be displayed */
+  disableReadtime?: boolean;
```

Comment:
i prefer to use `showReadtime` (false by default) instead of `disableReadtime`

Reply:
I prefer to make it even simpler:

```ts
/** Whether to display read time or not */
readTime?: boolean;
```

Reply:
Changed `disableReadtime` to `readtime`. Default is false / no readtime shown.